api/docs/release.dox

```diff
@@ -161,6 +161,8 @@ Further non-compatibility-affecting changes include:
    source and destination operands were reversed).
  - Fixed the VEX-encoded forms of vaesdec/vaesdeclast/vaesenc/vaesenclast/vpclmulqdq
    to obey VEX.L, which they previously ignored.
+ - Added instr_get_operation_size() and instr_set_operation_size() APIs for
```

Comment:
This is in the wrong place: we put out 11.1 so this list can't be changed.  This has to be up on the post-11.1 list.